release/release_gold.sh

```diff
@@ -420,6 +420,12 @@ function reference_new_releases {
 				--json "url" \
 				--repo "pyoo47/liferay-jenkins-ee")"
 
+		git checkout master
+
+		git branch -D "${_TEMP_BRANCH}"
+
+		git push "git@github.com:liferay-release/liferay-jenkins-ee.git" --delete "${_TEMP_BRANCH}"
```

Comment:
@kiwm, can you please create a function inside `_git.sh` for this code for we reuse it?